packages/manager/modules/pci/src/projects/project/kubernetes/details/controller.js

```diff
@@ -1,9 +1,11 @@
-import { DOC_URL } from './constants';
+import { DOC_URL, TRACKING_TABS } from './constants';
 
 export default class KubernetesCtrl {
   /* @ngInject */
-  constructor($scope) {
+  constructor($scope, $state) {
     this.$scope = $scope;
+    this.$state = $state;
```

Comment:
Why do you need `$state` ?

Reply:
I used it during the dev process but it is indeed useless in the end, I remove it.